src/actions/index.js

```diff
@@ -0,0 +1,29 @@
+import LocalApi from "./../apis/LocalApi";
+
+export const setImages = images => {
+  return {
+    type: "SET_IMAGES",
+    payload: images
+  };
+};
+
+export const fetchImages = () => {
+  return async (dispatch, getState) => {
+    let response = await LocalApi.get("/images");
+    return dispatch({
+      type: "SET_IMAGES",
+      payload: response.data
+    });
+  };
+};
+
+//[Use later? - Unable to use redux to create apartments]

```

Comment:
But unable to use Redux to create apartments. I'll research more on this later :)